react/containers/ShowChannel/view.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ErrorPage from 'components/ErrorPage';
+import NavBar from 'containers/NavBar';
+import ChannelClaimsDisplay from 'containers/ChannelClaimsDisplay';
+
+import { CHANNEL } from 'constants/show_request_types';
+
+function requestIsAChannelRequest ({ requestType }) {
+  return requestType === CHANNEL;
+}
+
+class ShowChannel extends React.Component {
+  componentDidMount () {
+    const { previousRequest, channel, requestId, requestChannelName, requestChannelId } = this.props;
+    if (!previousRequest) {
```

Comment:
Same comments above about `previousRequest`. I think a more understandable approach would just be:
```
if (!channel) this.props.onNewChannelRequest(...)
```

Reply:
See above re: previous request.  I mostly fixed this, but not sure if more consolidation can be done.